tests/functional.mk

```diff
@@ -8,8 +8,21 @@ FTST_IMAGE := $(FTST_TMPDIR)/rkt-inspect.aci
 FTST_IMAGE_MANIFEST_SRC := $(MK_SRCDIR)/image/manifest
 FTST_IMAGE_MANIFEST := $(FTST_IMAGE_DIR)/manifest
 FTST_IMAGE_TEST_DIRS := $(FTST_IMAGE_ROOTFSDIR)/dir1 $(FTST_IMAGE_ROOTFSDIR)/dir2
+FTST_ACE_MAIN_IMAGE_DIR := $(FTST_TMPDIR)/ace-main
+FTST_ACE_MAIN_IMAGE_ROOTFSDIR := $(FTST_ACE_MAIN_IMAGE_DIR)/rootfs
```

Comment:
`FTST_ACE_MAIN_IMAGE_ROOTFSDIR` and `FTST_ACE_SIDEKICK_IMAGE_ROOTFSDIR` can be removed. The `FTST_GENERATE_ACE_IMAGE` takes care of creating them.